bedrock/products/templates/products/vpn/landing.html

```diff
@@ -312,7 +312,9 @@ <h3>{{ ftl('vpn-landing-faq-compatibility-question-heading') }}</h3>
           {{ ftl('vpn-landing-faq-compatibility-question-desc-windows-v3', fallback='vpn-landing-faq-compatibility-question-desc-windows-v2', url=url('products.vpn.platforms.windows')) }}
         </li>
         <li>
-          {% if ftl_has_messages('vpn-landing-faq-compatibility-question-desc-mac-v3') %}
+          {% if ftl_has_messages('vpn-landing-faq-compatibility-question-desc-mac-v4') %}
+            {{ ftl('vpn-landing-faq-compatibility-question-desc-mac-v4', url=url('products.vpn.platforms.mac')), minversion='10.15' }}
```

Comment:
Parentheses need fixing:

```suggestion
            {{ ftl('vpn-landing-faq-compatibility-question-desc-mac-v4', url=url('products.vpn.platforms.mac'), minversion='10.15') }} minversion='10.15' }}
```